modules/Bio/EnsEMBL/PipeConfig/VariationMart_conf.pm

```diff
@@ -0,0 +1,603 @@
+                genomic_features_dbname => $self->o('genomic_features_dbname'),
+                division_name           => $self->o('division_name'),
+            },
+            -max_retry_count   => 0,
+            -analysis_capacity => 10,
+            -can_be_empty      => 1,
+            -flow_into         => [ 'AnalyzeTables' ],
+        },
+
+        {
+            -logic_name      => 'AnalyzeTables',
+            -module          => 'Bio::EnsEMBL::VariationMart::AnalyzeTables',
+            -parameters      => {
+                optimize_tables => $self->o('optimize_tables'),
+            },
+            -max_retry_count => 0,
+            -flow_into       => [ 'run_tests' ],
+        },
+        {
+            -logic_name => 'run_tests',
+            -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+            -parameters => {
+                cmd         => 'cd #test_dir#;perl #base_dir#/ensembl-biomart/scripts/pre_configuration_mart_healthcheck.pl -newuser #user# -newpass #pass# -newport #port# -newhost #host# -olduser #olduser# -oldport #oldport# -oldhost #oldhost# -new_dbname #mart# -old_dbname #old_mart# -old_rel #old_release# -new_rel #new_release# -empty_column 1 -grch37 #grch37# -mart snp_mart',
+                mart        => $self->o('mart_db_name'),
+                user        => $self->o('user'),
+                pass        => $self->o('pass'),
+                host        => $self->o('host'),
+                port        => $self->o('port'),
+                olduser     => $self->o('olduser'),
+                oldhost     => $self->o('oldhost'),
+                oldport     => $self->o('oldport'),
+                old_mart    => $self->o('old_mart'),
+                test_dir    => $self->o('test_dir'),
+                old_release => $self->o('old_release'),
+                new_release => $self->o('ensembl_release'),
+                base_dir    => $self->o('base_dir'),
+                grch37      => $self->o('grch37'),
+            },
+            -flow_into  => [ 'check_tests' ],
+        },
+        {
+            -logic_name => 'check_tests',
+            -module     => 'Bio::EnsEMBL::Hive::RunnableDB::SystemCmd',
+            -parameters => {
+                cmd      => 'EXIT_CODE=0;failed_tests=`find #test_dir#/#old_mart#_#oldhost#_vs_#mart#_#host#.* -type f ! -empty -print`;if [ -n "$failed_tests" ]; then >&2 echo "Some tests have failed please check ${failed_tests}";EXIT_CODE=1;fi;exit $EXIT_CODE',
+                mart     => $self->o('mart_db_name'),
+                host     => $self->o('host'),
+                oldhost  => $self->o('oldhost'),
+                old_mart => $self->o('old_mart'),
+                test_dir => $self->o('test_dir'),
+            },
+        }
+    ];
+}
+
+sub resource_classes {
+    my ($self) = @_;
+    return {
+        'default'  => { 'LSF' => '-q production' },
+        'normal'   => { 'LSF' => '-q production -M 4096 -R "rusage[mem=4096]"' },
```

Comment:
As above, maybe change name to '4Gb_mem' for consistency/clarity?

Reply:
Moved to 4Gb_mem for consistency, well spotted.